packages/react-integration/demo-app-ts/src/components/AppHeader/AppHeader.tsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import { Avatar, Brand, PageHeader } from '@patternfly/react-core';
 import { AppToolbar } from '../AppToolbar/AppToolbar';
-const imgBrand = '../../assets/images/imgBrand.svg';
-const imgAvatar = '../../assets/images/imgAvatar.svg';
+const imgBrand = require('../../assets/images/imgBrand.svg');
```

Comment:
not a fan of seeing require statements here. How about an alternate approach:

1. create a `custom.d.ts` file in the demo-app-ts dir.
2. content:
```
declare module '*.svg' {
  const content: string;
  export default content;
}
```
3. Add to the tsconfig.json file
`"files": [ "custom.d.ts" ]`

4. Now you can use
`import imgBrand from '../../assets/images/imgBrand.svg'`

Reply:
From my point of view that seems to introduce more "entropy", for a lack of a better word. But I'll refactor it.